src/Components/Admins/form.jsx

```diff
@@ -0,0 +1,177 @@
+import { useState, useEffect } from 'react';
+import styles from './admins.module.css';
+
+const params = new URLSearchParams(window.location.search);
+const adminId = params.get('id');
+
+const AdminsForm = () => {
+  const [inputValue, setInputValue] = useState({
+    name: '',
+    lastName: '',
+    email: '',
+    password: '',
+    status: false
+  });
+
+  useEffect(async () => {
+    if (adminId !== null) {
+      try {
+        const response = await fetch(`${process.env.REACT_APP_API_URL}/admins/${adminId}`);
+        const data = await response.json();
+        setInputValue({
+          name: data.data.name,
+          lastName: data.data.lastName,
+          email: data.data.email,
+          password: data.data.password,
+          status: data.data.status
+        });
+      } catch (err) {
+        alert('Admin not found');
+      }
+    }
+  }, []);
+
+  const onChangeInput = (e) => {
+    setInputValue({ ...inputValue, [e.target.name]: e.target.value });
+  };
+
+  const redirect = () => {
+    window.location.assign('/admins');
+  };
+
+  const onSubmit = () => {
+    if (adminId !== null) {
+      inputValue.status = inputValue.status === 'active' ? true : false;
+      const putOptions = {
+        method: 'PUT',
+        headers: {
+          'Content-Type': 'application/json'
+        },
+        body: JSON.stringify(inputValue)
+      };
+      const url = `${process.env.REACT_APP_API_URL}/admins/${adminId}`;
+      fetch(url, putOptions).then(async (response) => {
+        if (response.status !== 200 && response.status !== 201) {
+          const { message } = await response.json();
+          alert(message);
+        } else {
+          alert('Admin updated successfully');
```

Comment:
```suggestion
          alert(message);
```
Here you can use the message from the backend

Reply:
excelent correction. Done